standard-format.md

```diff
@@ -0,0 +1,89 @@
+# Understanding the format of standard
+
+This library depends on a "standard" which classifies a certain change as `breaking`, `non-breaking` or `unclassified`.
+
+This object has the following structure:
+
+```
+{
+	[jsonPointerPath]: {
+		add: 'breaking' | 'non-breaking' | 'unclassified',
+		remove: 'breaking' | 'non-breaking' | 'unclassified',
+		edit: 'breaking' | 'non-breaking' | 'unclassified',
+	}
+}
+```
+
+`jsonPointerPath` is a path that points to a certain property from the AsyncAPI specification.
+
+Example:
+
+```yml
+info:
+  license:
+    name: Apache 2.0
+```
+
+The jsonPointer to the `name` will be `/info/license/name`.
+
+And each `jsonPointerPath` will have 3 properties. `add`, `remove` and `edit`, these points to three operations that can occur when modifying a AsyncAPI document.
+
+1. `add` - A new property has been added.
+2. `remove` - An existing property has been deleted.
+3. `edit` - An existing property has been modified.
+
+and these 3 properties will have any one of the acceptable values.
```

Comment:
you need to rephrase as I do not know what you mean

Reply:
Rephrased it. Can you check?

Reply:
lgtm 🚀